day2/Day 2.py

```diff
@@ -0,0 +1,24 @@
+# Storing the letters of the alphabet into a variable
```

Comment:
When working on day 2, switch back to `master` first, then `create new branch` and work/commit/push your day2 stuff to that branch. Now you've got a PR with 2 'features'